menu/menu_setting.c

```diff
@@ -17860,6 +17925,164 @@ static bool setting_append_list(
          END_SUB_GROUP(list, list_info, parent_group);
          END_GROUP(list, list_info, parent_group);
          break;
+#if defined(HAVE_NETWORKING) && defined(HAVE_CLOUD_STORAGE)
+      case SETTINGS_LIST_CLOUD_STORAGE:
+         START_GROUP(list, list_info, &group_info,
+               msg_hash_to_str(MENU_ENUM_LABEL_VALUE_CLOUD_STORAGE_SETTINGS),
+               parent_group);
+
+         parent_group = msg_hash_to_str(MENU_ENUM_LABEL_VALUE_CLOUD_STORAGE_SETTINGS);
+
+         START_SUB_GROUP(list, list_info,
+               "Cloud Storage",
+               &group_info, &subgroup_info, parent_group);
+
+         CONFIG_BOOL(
+               list, list_info,
+               &settings->bools.cloud_storage_enable,
+               MENU_ENUM_LABEL_CLOUD_STORAGE_ENABLE,
+               MENU_ENUM_LABEL_VALUE_CLOUD_STORAGE_ENABLE,
+               false,
+               MENU_ENUM_LABEL_VALUE_OFF,
+               MENU_ENUM_LABEL_VALUE_ON,
+               &group_info,
+               &subgroup_info,
+               parent_group,
+               general_write_handler,
+               general_read_handler,
+               SD_FLAG_NONE);
+         (*list)[list_info->index - 1].action_ok     = &setting_bool_action_left_with_refresh;
```

Comment:
Maybe we want to split this macro up https://github.com/libretro/RetroArch/blob/367d7850852dc985d233f048e330f56958979b88/menu/menu_setting.c#L246

Reply:
Changed to use this macro. I see that it is assuming variables in the current scope.